ui/src/test/org/jboss/windup/web/selenium/Selenium01Test.java

```diff
@@ -0,0 +1,198 @@
+package org.jboss.windup.web.selenium;
+
+import java.awt.AWTException;
+import junit.framework.TestCase;
+
+/**
+ * Runs the first test from the Web UI Test Script V0.1
+ * @author elise
+ */
+public class Selenium01Test extends TestCase {
+
+	private CreateProject selenium;
+
+	public void setUp() {
+		selenium = new CreateProject();
+	}
+
+	/**
+	 * Tests the first and second step
+	 */
+	public void testStep01_02() {
+		/*
+		 * Step 01
+		 */
+		assertEquals("http://127.0.0.1:8080/rhamt-web/project-list", selenium.checkURL());
+		
+		selenium.clickNewProjButton();
+		assertEquals("http://127.0.0.1:8080/rhamt-web/wizard/create-project", selenium.checkURL());
+		
+		//checks that the project name field has focus, then the cancel/next buttons are enabled/disabled
+		//.nameInputSelected does not work
+		assertTrue(selenium.nameInputSelected());
+		assertTrue(selenium.cancelEnabled());
+		assertFalse(selenium.nextEnabled());
+		
+		/*
+		 * step 02
+		 */
+		// have to click cancel twice to get out of the name input
+		selenium.clickCancel();
+		selenium.clickCancel();
+		
+		assertEquals("http://127.0.0.1:8080/rhamt-web/project-list", selenium.checkURL());
+		
+		selenium.closeDriver();
+	}
+
+	/**
+	 * checks the third step
+	 */
+	public void testStep03_00() {
+		assertEquals("http://127.0.0.1:8080/rhamt-web/project-list", selenium.checkURL());
+		selenium.clickNewProjButton();
+		assertEquals("http://127.0.0.1:8080/rhamt-web/wizard/create-project", selenium.checkURL());
+
+		//checks for next being enabled after entering in 3 characters
+		selenium.inputProjName("abc");
+		assertTrue(selenium.nextEnabled());
+		selenium.clearProjName();
+		assertFalse(selenium.nextEnabled());
+
+		//properly inputs the project name & description
+		selenium.inputProjName("test");
+		assertTrue(selenium.nextEnabled());
+		selenium.inputProjDesc("for the selenium test"); 
+		
+		//checks that it redirects to the correct page
+		selenium.clickNext();
+		//assertEquals("/add-applications", selenium.checkURL().substring(51));
+		
+		//checks that the upload panel is active & the next button is enabled
+		assertEquals("Upload", selenium.activePanel());
+		assertFalse(selenium.nextEnabled());
+
+	}
+
+	/**
+	 * checks the 4th through 7th step 
+	 * @throws AWTException for the robot
+	 */
+	public void testStep04_07() throws AWTException {
+		testStep03_00();
+
+		/*
+		 * Step 04
+		 */
+		assertFalse(selenium.nextEnabled());
+		selenium.clickChooseFiles();
+		
+		/*
+		 * Step 05
+		 */
+		selenium.robotCancel();
+		//checks that the user has been returned to the correct page
+		assertTrue(selenium.checkURL().endsWith("/add-applications"));
+		//checks that there are no files pulled up
+		assertTrue(selenium.voidFile());
+
+		/*
+		 * Step 06
+		 */
+		selenium.clickChooseFiles();
+		//AdministracionEfectivo.ear
+		String s = "/home/mbrophy/Sample_Files/06__all_apps/01/AdministracionEfectivo.ear";
```

Comment:
There are different references to files just like this one and they aren't going to work.
I'm not reporting this for every occurrence in this PR but they need to be fixed.

Reply:
Is there a standard path where sample or test files can be found on the CI server?

Reply:
First we've better focusing on ensure that everyone can clone this windup-web project and run the tests without the need for any external resource.
For example, the `services` module has this example application [sample-tiny.war](https://github.com/windup/windup-web/blob/master/services/src/test/resources/sample/sample-tiny.war) in  `services/src/test/resources/sample/` folder for testing.
The `windup` project has the [jee-example-app-1.0.0.ear](https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear) example application that we also distribute as sample application in CLI and Web UI releases.
I think you could add something like a `test-archives` folder inside [`ui/src/test/resources`](https://github.com/windup/windup-web/tree/master/ui/src/test/resources) path where to put some "light weight" test application just like `jee-example-app-1.0.0.ear`

Reply:
comitted changes to make test-files self-contained within ui project